.changeset/nice-radios-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"10up-toolkit": patch
+---
+
+Improve Sass compatibility by making sure PostCSS runs after Sass has finished and also ensuring that PostCSS process the Sass pipeline
```

Comment:
Can you include what you're fixing in the changeset?

Reply:
@nicholasio you mean the actual issues or more information around the changes?

Reply:
The Github issues